packages/nimble-components/src/wafer-map/specs/README.md

```diff
@@ -226,8 +232,8 @@ N/A
 
 ### Performance
 
-As mentioned in the "Risks" section, the biggest challenge from the performance standpoint is the representation of large number of datapoints and interacting with these datapoints in the browser.
-Currently the only thing we would like to achieve is to measure (benchmark) the time it takes a `nimble-wafer-map` component to be loaded with various number of datapoints. Initially this could be a manual benchmark on a demo site hosted within the repository. Longer term we may automate and trend this by including benchmarking on the PR build.
+As mentioned in the "Risks" section, the biggest challenge from the performance standpoint is the representation of large number of data points and interacting with these data points in the browser.
```

Comment:
This feature, as I understood it, (and also from the title of this PR) does not address performance issues. I am not sure it makes sense to refer here to performance if no performance improvements/changes are actually expected from this feature ... 

If the idea is to add bench-marking, as part of the storybook rendering, then the title should reflect that. However, again, if the intent is to do a "manual" benchmark using browser tools, I am not convinced it should be treated or otherwise included in the HLD ...

Reply:
The slight performance improvement this change will create is related to the internal parsing of the data for finding the bounding box, which I think can be more apparent on larger data sets.

I added this section with some actual benchmark data because of a concern raised by @rajsite that the wafer documentation was not clear which data volume it was comfortable handling, where the current limit is and if we are planning on improving in the near to medium future.